uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader.php

```diff
@@ -188,6 +189,28 @@ private function createDefaultRelationshipTypes() {
     }
   }
 
+  /**
+   * Making All Currencies Available for new installations
+   */
+  private function makeAllCurrenciesAvailable() {
+    $dao = CRM_Core_DAO::executeQuery('SELECT * from civicrm_currency');
```

Comment:
Please move this line to just before the while loop.
You can also add a space before that so the code is not cluttered together